include/mfmg/dealii/multivector.hpp

```diff
@@ -0,0 +1,54 @@
+/*************************************************************************
+ * Copyright (c) 2017-2019 by the mfmg authors                           *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * This file is part of the mfmg libary. mfmg is distributed under a BSD *
+ * 3-clause license. For the licensing terms see the LICENSE file in the *
+ * top-level directory                                                   *
+ *                                                                       *
+ * SPDX-License-Identifier: BSD-3-Clause                                 *
+ *************************************************************************/
+
+#ifndef MFMG_MULTIVECTOR_HPP
+#define MFMG_MULTIVECTOR_HPP
+
+#include <mfmg/common/exceptions.hpp>
```

Comment:
You need to include `<algorithm>` and `<vector>`
I am not mentioning `<memory>` cause I sincerely hope we are not going to introduce more (sigh...) shared pointers